ci/python3.13.yaml

```
@@ -0,0 +1,8 @@
+channel_sources:
+- conda-forge,defaults
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.13.* *_cpython
```

For complicated reasons, they decided to name this differently for python 3.13:
```suggestion
- 3.13.* *_cp313
```